pkg/sql/sqlclustersettings/clustersettings.go

```diff
@@ -129,7 +129,9 @@ var LDRImmediateModeWriter = settings.RegisterStringSetting(
 	settings.ApplicationLevel,
 	"logical_replication.consumer.immediate_mode_writer",
 	"the writer to use when in immediate mode",
-	metamorphic.ConstantWithTestChoice("logical_replication.consumer.immediate_mode_writer", string(LDRWriterTypeSQL), string(LDRWriterTypeLegacyKV), string(LDRWriterTypeCRUD)),
+	// TODO(jeffswenson): make the sql writer the default.
+	//metamorphic.ConstantWithTestChoice("logical_replication.consumer.immediate_mode_writer", string(LDRWriterTypeSQL), string(LDRWriterTypeLegacyKV), string(LDRWriterTypeCRUD)),
```

Comment:
why remove the metamorphic wrapper with the other modes? Perhaps this comment above was temporary.

Reply:
This is temporary until we re-fix the sql writer tombstone handling.

Reply:
sg